frontend/src/metabase/query_builder/components/dataref/ConnectedTableList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { jt } from "ttag";
+
+import Table from "metabase-lib/lib/metadata/Table";
+
+import {
+  NodeListItem,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+} from "./NodeList.styled";
+
+type Props = {
+  tables: Table[];
+  onTableClick: (table: Table) => void;
+};
+
+function ConnectedTableList({ tables, onTableClick }: Props) {
```

Comment:
nit: We define components with `const`, e.g. `const ConnectedTableList = ({ tables }) => {`